app/(dashboard)/settings/tools/apiForm.tsx

```
@@ -124,7 +130,11 @@ const ApiForm = ({ onCancel }: ApiFormProps) => {
         <Button variant="ghost" onClick={onCancel} disabled={importMutation.isPending}>
           Cancel
         </Button>
-        <Button variant="bright" onClick={handleImport} disabled={importMutation.isPending}>
+        <Button
+          variant="bright"
+          onClick={handleImport}
+          disabled={importMutation.isPending}
+        >
```

ESLint

[error] Replace `⏎··········variant="bright"⏎··········onClick={handleImport}⏎··········disabled={importMutation.isPending}⏎········` with `·variant="bright"·onClick={handleImport}·disabled={importMutation.isPending}`

(prettier/prettier)